.pre-commit-config.yaml

```diff
@@ -96,7 +96,7 @@ repos:
         name: Spell check with cspell
 
   - repo: https://github.com/jsh9/pydoclint
-    rev: "0.5.12"
+    rev: "0.5.9"
```

Comment:
is the version meant to be downgraded? The PR title does not match the change.

Reply:
Haha! Clearly no. Interesting bit is that locally I no longer can run pydoclint due to the raised bug and I tried on two different machines: macos and ubuntu, both arm64 and using python installed using mise.